metadata/990__30__metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "form_990",
```

Comment:
This should be removed. Isn't it @andznui ?

Reply:
removed it. Also removed 24.10.0 version